app/views/user_mailer/approved_borrower.html.erb

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html>
+  <head>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type" />
+  </head>
+  <body>
+    <h1>Welcome to keevahh.com, <%= @user.first_name %></h1>
```

Comment:
Maybe change this to "Congratulations, <%= @tenant.organization %> was approved on keevahh". Or something along those lines.